extern/config.mad.in.h

```diff
@@ -87,44 +86,53 @@
 /* #undef OPT_STRICT */
 
 /* Name of package */
-#define PACKAGE "libmad"
+/* #undef PACKAGE */
 
 /* Define to the address where bug reports for this package should be sent. */
-#define PACKAGE_BUGREPORT "support@underbit.com"
+/* #undef PACKAGE_BUGREPORT */
 
 /* Define to the full name of this package. */
-#define PACKAGE_NAME "MPEG Audio Decoder"
+/* #undef PACKAGE_NAME */
 
 /* Define to the full name and version of this package. */
-#define PACKAGE_STRING "MPEG Audio Decoder 0.15.1b"
+/* #undef PACKAGE_STRING */
 
 /* Define to the one symbol short name of this package. */
-#define PACKAGE_TARNAME "libmad"
+/* #undef PACKAGE_TARNAME */
 
 /* Define to the version of this package. */
-#define PACKAGE_VERSION "0.15.1b"
+/* #undef PACKAGE_VERSION */
 
 /* The size of a `int', as computed by sizeof. */
-#define SIZEOF_INT 4
+#cmakedefine SIZEOF_INT ${SIZEOF_INT}
 
 /* The size of a `long', as computed by sizeof. */
-#define SIZEOF_LONG 4
+#cmakedefine SIZEOF_LONG ${SIZEOF_LONG}
 
 /* The size of a `long long', as computed by sizeof. */
-#define SIZEOF_LONG_LONG 8
+#cmakedefine SIZEOF_LONG_LONG ${SIZEOF_LONG_LONG}
 
 /* Define to 1 if you have the ANSI C header files. */
-#define STDC_HEADERS 1
+#cmakedefine STDC_HEADERS 1
 
 /* Version number of package */
-#define VERSION "0.15.1b"
+/* #undef VERSION */
+
+/* Define to 1 if your processor stores words with the most significant byte
+   first (like Motorola and SPARC, unlike Intel and VAX). */
+#cmakedefine WORDS_BIGENDIAN 1
 
 /* Define to empty if `const' does not conform to ANSI C. */
 /* #undef const */
 
-/* Define as `__inline' if that's what the C compiler calls it, or to nothing
-   if it is not supported. */
-#define inline __inline
+/* Define to `__inline__' or `__inline' if that's what the C compiler
+   calls it, or to nothing if 'inline' is not supported under any name.  */
+/* #ifndef __cplusplus
+#undef inline
+#endif */
 
 /* Define to `int' if <sys/types.h> does not define. */
-/* #undef pid_t */
+#cmakedefine HAVE_SIZEOF_PID_T 1
+#if !defined(HAVE_SIZEOF_PID_T)
+typedef int pid_t;
+#endif
```

Comment:
I have `sys/types.h` as a header. However, I still do not have `pid_t` defined. This is the appropriate check. ...I'll admit the comment up above should have been modified in retrospect, but it will work nonetheless.